Modules/ThirdParty/Eigen3/src/itkeigen/CMakeLists.txt

```diff
@@ -753,15 +840,14 @@ add_library (ITKInternalEigen3::Eigen ALIAS eigen_internal)
 # to avoid introducing GPL code from Eigen3 internally in ITK.
 option(ITK_USE_EIGEN_MPL2_ONLY "Set compile definition EIGEN_MPL2_ONLY for ITKInternalEigen3." OFF)
 mark_as_advanced(ITK_USE_EIGEN_MPL2_ONLY)
+option(ITK_EIGEN_MANGLE "Replace Eigen namespace with itkeigen to avoid conflicts with other Eigen versions." OFF)
+mark_as_advanced(ITK_EIGEN_MANGLE)
```

Comment:
```suggestion
```

Since the mangling is based on the include directory in which Eigen can found, I don't think we need this option here.